washboard-ui/src/services/lattice/index.ts

```diff
@@ -0,0 +1,6 @@
+export {useLatticeData} from './hooks/use-lattice-data';
+export {useLatticeConfig} from './hooks/use-lattice-config';
+export {canConnect} from './helpers/can-connect';
+export {LatticeServiceProvider} from './context/LatticeServiceProvider';
+export {LatticeService} from './classes/lattice-service';
+export type {WadmActor, WadmHost, WadmLink, WadmProvider} from './classes/lattice-service';
```

Comment:
At the app-level code, avoid re-exporting, this becomes an anti-pattern. At the app-level, go directly to where things are define.

Reply:
Normally I would agree, but this is what it's going for:
> Add single barrel export for Lattice service "package"

Eventually, this will be it's own package and having a single main export means that there's a clear API for using it